src/Text/Layout/Table/Spec/HeaderColSpec.hs

```diff
@@ -16,4 +16,8 @@ headerColumn = HeaderColSpec
 
 -- | Header columns are usually centered.
 instance Default HeaderColSpec where
-    def = headerColumn center def
+    def = defHeaderColSpec
+
+-- | The default 'HeaderColSpec' centres the text and uses no 'CutMark'.
```

Comment:
Are you British? :laughing:

Reply:
Canadian–Australian. Should I use the American spelling?